### ISSUE TITLE
Introduce contextual tool tips

### ISSUE THREAD

**Issue by pippinsplugins** [restrictcontentpro/restrict-content-pro#518]
To help reduce support, we should introduce inline, contextual tool tips throughout the plugin.

**Comment by pippinsplugins** [comment]
Being done partly through #516. Will continue once that is merged into `release/2.6`.

**Comment by pippinsplugins** [comment]
This is ready for review @michaelbeil and @mindctrl

**Comment by mindctrl** [comment]
This is what I see by default now:

![image](https://cloud.githubusercontent.com/assets/1254870/16381650/8f43793a-3c4b-11e6-9ff5-42417b58f5f3.png)

Didn't we intend on this showing this instead?

![image](https://cloud.githubusercontent.com/assets/1254870/16381661/9dd8a862-3c4b-11e6-8df0-173d9aafab25.png)

**Comment by mindctrl** [comment]
Oops commented on wrong issue.

**Comment by mindctrl** [comment]
The tooltip feature is working great here.

**Comment by pippinsplugins** [comment]
All text look good? Any missing tooltips?

**Comment by mindctrl** [comment]
@pippinsplugins On the `Recurring` tooltip on member edit screen:

Is it also worth mentioning the obvious, which is "This member has a recurring subscription.". So the whole thing might look something like this:

`This member has a recurring subscription. Only customers with recurring memberships will be given the option to cancel their membership on their subscription details page.`

Also added the additional text at the end to clarify where they'll see the option. Thoughts? Otherwise I'm good with everything else.

**Comment by pippinsplugins** [comment]
Updated but added `If checked` to the front.

**Comment by mindctrl** [comment]
Looks good.

**Comment by pippinsplugins** [comment]
Merge conflicts with `release/2.6` were nasty. Can you give it another once over?

**Comment by mindctrl** [comment]
@pippinsplugins I see some leftover merge stuff at very top of `includes/css/admin-styles.css`. Otherwise looks good.

**Comment by mindctrl** [comment]
There's another instance of `update_user_meta()` missing the third parameter in `set_renewed_date()`.

**Comment by pippinsplugins** [comment]
Fixed.
